### ISSUE TITLE
Mistake in grid/01-grid-layout-1/README.md

### ISSUE THREAD

**Issue by TobiOkanlawon** [TheOdinProject/css-exercises#136]
- [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
- [x] The title of this issue is similar to the `file, exercise/lesson, or folder: brief description of bug` format, e.g. `React section: Knowledge Checks don't link to resource`

- [x] I would like to be assigned this issue to work on it

**1. Description of the Bug**

The line in the README.md file says: "- The first column is three times larger than the other".
In reality, according to the desired-outcome.png file, the second column is bigger than the other.

**2. How To Reproduce**

Open the README.md file and read the README.md file (line 19) :)

**3. Expected Behavior**

The line in the README.md file should say: "- The second column is three times larger than the other".

**4. Desktop/Device:**
 - Device: 
 - OS: 
 - Browser: 
 - Version: 

**5. Additional Information**

I would change it myself, and really it isn't that big of a problem, but I haven't learnt all about git yet and I don't want to take a bigger detour than I currently am. But I'd like to contribute, sometime.

In other words, I am not yet confident in my abilities (for some reason, I mean it's probably quite easy to do). In fact, I'll fix it.

**Comment by TobiOkanlawon** [comment]
I'd like to fix it please.

**Comment by dm-murphy** [comment]
Thanks Tobi, but this was previously fixed in a PR from 20 days ago. If you are still seeing the old version you can update your repo.
